### DISCUSSION TITLE
container password

### DISCUSSION THREAD

**Discussion by nicomarti83** [haugene/docker-transmission-openvpn#2118] [Q&A] [answered]
Hi,
I'm tring to resolve recent problems with my container (last version) and I need to log into it, but for the first time I need a password to enter... Someone know it?
thanks in advance

**Comment by ilike2burnthing** [comment, marked as answer]
Check what you've set for your username and password - https://haugene.github.io/docker-transmission-openvpn/faq/#how_do_i_enable_authentication_in_the_web_ui

**Reply by nicomarti83** [reply]
I try to SSH into the container with the command `sudo docker exec -it "container name" /bin/bash` the password I've set for connecting to RPC does not work... I gess there is a master password

**Reply by pkishino** [reply]
No, there is no master password or anything such like. The only password is the one you set. You can always delete the container and recreate it. It will read the password you set

**Comment by nicomarti83** [comment]
Ok that was not helping me but for the record the password asked is my qnap admin password... 


 [xxxxxx@xxxxxx` ~]$ sudo docker exec -it nord /bin/bash                  

We trust you have received the usual lecture from the local System
Administrator. It usually boils down to these three things:

    #1) Respect the privacy of others.
    #2) Think before you type.
    #3) With great power comes great responsibility.

Password: 
<

**Reply by pkishino** [reply]
Sigh, another nonsense NAS annoyance. Glad you figured it out though
